core/dutydb/memory.go

```diff
@@ -56,8 +63,11 @@ func (db *MemDB) Store(_ context.Context, duty core.Duty, unsignedSet core.Unsig
 
 	switch duty.Type {
 	case core.DutyProposer:
-		for pubkey, unsignedData := range unsignedSet {
-			err := db.storeBeaconBlockUnsafe(pubkey, unsignedData)
+		if len(unsignedSet) > 1 {
```

Comment:
add a comment on what does this if condition mean

Reply:
```suggestion
		// Sanity check max one proposer per slot 
		if len(unsignedSet) > 1 {
```